src/Umbraco.Web.UI.Client/src/views/propertyeditors/mediapicker/mediapicker.html

```diff
@@ -7,6 +7,7 @@
         <ul ui-sortable="sortableOptions" ng-model="images" class="umb-sortable-thumbnails">
             <li data-element="sortable-thumbnail-{{$index}}" class="umb-sortable-thumbnails__wrapper" ng-repeat="image in images track by $index">
 
+                <span class="label trashed" ng-if="image.trashed">trashed</span>
                 <!-- IMAGE -->
                 <img ng-class="{'trashed': image.trashed}" ng-src="{{image.thumbnail}}" alt="" ng-show="image.thumbnail" title="{{image.trashed ? 'Trashed: ' + image.name : image.name}}" />

```

Comment:
Why is the class applied to the image ? Don't see any specific styling for that

Reply:
@dawoe this control can pick two different types, either images or file types (e.g. PDF) 
When you are picking images, the image.thumbnail (visibility toggled by ng-show) is used and the trashed class applies an opacity to the thumbnail (that's prior to this PR)
My PR handles the scenario when files are picked (which don't have a thumbnail) and reuses the same class name

Reply:
![u4-11152-before-after](https://user-images.githubusercontent.com/78284/42121103-df89f550-7c7b-11e8-9396-118de010dc68.png)

This screenshot shows before & after